layouts/_default/single.html

```diff
@@ -1,5 +1,5 @@
 {{ define "main" }}
 <h1>{{ .Title }}</h1>
-{{ with .Lastmod }}<p class="last-modified">Last modified: {{ . | time.Format ":date_medium" }}</p>{{ end }}
+<p class="last-modified">{{ .Param "modificated" }} {{ with .Lastmod }}{{ . | time.Format ":date_medium" }}</p>{{ end }}
```

Comment:
Here you moved the `{{ with .Lastmod }}` block inwards, causing half of the last mod paragraph to always be included. It should be before the paragraph element, so it either includes the whole localized paragraph or not at all if the lastmod attribute is not defined.

Reply:
I'll have to look a bit more into this, as I changed it because otherwise the translation variable would work. It does make sense like that, but as currently the site only has a single page that does contain the lastmod tag, I didn't think about that.